.wuf/required-component-files.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Why is this file needed?
I'm asking to understand how this works

Reply:
this file describes the shape of components, aka the convention.

`wuf update` reads this file, then goes through file system and looks
for folders that look like this shape.

folders that look similar are considered components and are saved in `.wuf/components.json` file.

then, `.wuf/components.json` file is used in `wuf export-testkits` to generate testkits exports file.

later on this file will be used for many more things:

1. storybook index
1. components exports file (something like src/index.js)
1. `code` section in autodocs (instead of currently manual `allComponents` part in `code({ components: allComponents })`)
1. auditing (what components do not adhere to conventions)

Reply:
later on this file could even be used to describe some of file contents (it's complex but doable)

for example, to ensure that testkit file exports testkit and a type
definition. In different library, maybe testkit file exports something
else, so there's a place to set this shape